Mathlib/LinearAlgebra/BilinearForm/Hom.lean

```diff
@@ -34,23 +35,27 @@ Bilinear form,
 
 open BigOperators
 
+open LinearMap (BilinForm)
+
 universe u v w
 
 variable {R : Type*} {M : Type*} [CommSemiring R] [AddCommMonoid M] [Module R M]
 variable {R₁ : Type*} {M₁ : Type*} [CommRing R₁] [AddCommGroup M₁] [Module R₁ M₁]
 variable {V : Type*} {K : Type*} [Field K] [AddCommGroup V] [Module K V]
 variable {B : BilinForm R M} {B₁ : BilinForm R₁ M₁}
 
+namespace LinearMap
+
 namespace BilinForm
 
 section ToLin'
 
 /-- Auxiliary definition to define `toLinHom`; see below. -/
 def toLinHomAux₁ (A : BilinForm R M) (x : M) : M →ₗ[R] R where
   toFun y := A x y
-  map_add' := A.bilin_add_right x
-  map_smul' c := A.bilin_smul_right c x
-#align bilin_form.to_lin_hom_aux₁ BilinForm.toLinHomAux₁
+  map_add' := map_add _
+  map_smul' := map_smul _
+#align bilin_form.to_lin_hom_aux₁ LinearMap.BilinForm.toLinHomAux₁
```

Comment:
I think you can replace this with `:= A x`!

Reply:
```
def toLinHomAux₁ (A : BilinForm R M) (x : M) : M →ₗ[R] R := A x
```
gives me
```
type mismatch
  (fun x y ↦ (A x) y) x
has type
  M → R : Type (max u_1 u_2)
but is expected to have type
  M →ₗ[R] R : Type (max u_2 u_1)
```

Reply:
I've changed the proof of `toLinHom` so that it no longer depends on `toLinHomAux₁`. However, this PR introduces a dependency on `toLinHomAux₁` in `BilinearForm/Orthogonal` so I can't remove it altogether yet.

Reply:
I found a solution using a local attribute

Reply:
Thanks.